apps/trading/pages/markets/trade-grid.tsx

```diff
@@ -31,57 +34,110 @@ interface TradeGridProps {
   market: Market_market;
 }
 
+export const TradeMarketHeader = ({ market }: TradeGridProps) => {
+  const [open, setOpen] = useState(false);
+  const candlesClose: string[] = (market?.candles || [])
```

Comment:
this is the component. named it `TradeMarketHeader` because it is the market header and lives inside `TradeGrid`, it is part of the `TradePanels`. maybe I should rename it to `MarketHeader` if it makes more sense. 
![image](https://user-images.githubusercontent.com/16125548/172138458-7e6b559c-3697-444b-baf8-808f592888e3.png)

Reply:
Apologies that wasn't clear. I was refering to 'candlesClose'.